plugins/inputs/aliyuncms/aliyuncms.go

```diff
@@ -438,7 +494,7 @@ func (s *AliyunCMS) prepareTagsAndDimensions(metric *Metric) {
 L:
 	for {
 		select {
-		case s.discoveryData = <-s.dt.dataChan:
+		case s.discoveryData = <-s.dt.DataChan:
```

Comment:
Can you please explain why you do need to export the channel!?

Reply:
Please have a look on my comment regarding the `discoveryTool` object

Reply:
I will switch it to non-exported